contracts/teller-network/OfferStore.sol

```diff
@@ -84,6 +87,7 @@ contract OfferStore is Stakable, SecuredFunctions, Proxiable {
         burnAddress = _burnAddress;
 
         basePrice = 0.01 ether;
+        maxOffers = 10;
```

Comment:
Why set it to 0 again here? Since it is already initialized at 10

Reply:
If you use the offerstore contract directly, maxOffers will be initialized by 10 
```
    uint public maxOffers = 10;
```

However in the case of proxyes, that initialization does not work because it will not be the OfferStore contract the one keeping the state, but the proxy, hence the need of an init() function that initializes any state variable.

Reply:
Thanks TIL